dotCMS/src/integration-test/java/com/dotcms/MainSuite1a.java

```diff
@@ -96,6 +96,7 @@
         FolderIntegrityCheckerTest.class,
         HostFactoryImplTest.class,
         BundleFactoryImplTest.class,
+        com.dotcms.experiments.business.ExperimentUrlPatternCalculatorIntegrationTest.class
```

Comment:
Do we need fully qualified class name?

Reply:
done https://github.com/dotCMS/core/pull/26586/commits/3b03fe7de9c28ae9bf5fbd50d743fa7917b39f10#diff-953cc43177b8253012086517e0481c194e38d7cf8e5ff2cda6624c0a3b96f7a7R100